### DISCUSSION TITLE
Armor locking

### DISCUSSION THREAD

**Discussion by Confvse** [Mijago/D2ArmorPicker#40] [Ideas]
A couple of friends and me love your app. But we would love to see a feature like armor locking. So we don't use our pve or pvp armor for other builds.

**Comment by Mijago** [comment]
Hi! I just implemented this today. It's available in version 2: https://mijago.github.io/D2ArmorPicker/#/v2

You can simply disable an armor piece from being used by clicking this icon:
![image](https://user-images.githubusercontent.com/3903469/133931536-7812f630-f69d-4338-b2c3-7df69d75338a.png)

Afterwards, you can enable it again by clicking it in the corresponding settings-section:
![image](https://user-images.githubusercontent.com/3903469/133931558-2e229b37-4956-45be-9003-9f244ca1013d.png)


Note: I am aware that the CSS of the masterwork overlay in the new settings section is broken, and I'll fix it soon.
